### ISSUE TITLE
NNShark: added files license clarification

### ISSUE THREAD

**Issue by jvuillaumier** [nnstreamer/nnshark#4]
Hello,

NNShark was forked from GstShark v0.6.1 according to repository history.
It has 2 license files inherited from GStreamer project:
COPYING (GPL v2) https://github.com/nnstreamer/nnshark/blob/master/COPYING
COPYING.LESSER (LGPL v2.1) https://github.com/nnstreamer/nnshark/blob/master/COPYING.LESSER

Since GstShark v0.6.1 fork, a number of source files have been added into NNShark project - list below.
Among those source files, some are explicitly stated a LGPL v2.1, and some others have no header so no explicit license/copyright.

As there are 2 licence files in the NNShark project, GPLv2 and LGPLv2.1, can you please clarify which license is attached to these new files. Adding a header in every file to explicitly document license and copyright would help.

Thank you.

**Added files in gst-shark**
source: git diff --diff-filter=A v0.6.1 c459816

**Files with licence LGPL v2.1 (c) RidgeRun:**
plugins/tracers/gstlive.c
plugins/tracers/gstlive.h
tests/check/gst-shark/gstliveprofiler.c
tests/check/gst-shark/gstliveunit.c

**Files with no Licence and no copyright:**
plugins/tracers/gstliveprofiler.c
plugins/tracers/gstliveprofiler.h
plugins/tracers/gstliveunit.c
plugins/tracers/gstliveunit.h
plugins/tracers/visualizeutil.c
plugins/tracers/visualizeutil.h
scripts/graphics/log_visualizer.py
tests/check/gst-shark/visualizeutil.c
tests/mocks/mockncurses.c

**Comment by myungjoo** [comment]
Yes.. we definitely need to rebase nnshark with the latest GstShark.
It will probably remove files marked as proprietary as well.
